code/modules/projectiles/guns/projectile/shotgun/bull.dm

```diff
@@ -1,6 +1,6 @@
 /obj/item/weapon/gun/projectile/shotgun/bull
 	name = "FS SG \"Bull\""
-	desc = "A Frozen Star double-barreled pump action shotgun. Marvel of engineering, this gun  often use by police SWAT teams."
+	desc = "A double-barreled pump action shotgun of \"Frosen Star\". Marvel of engineering, this gun is often use by SWAT teams."
```

Comment:
very wrong, you missspelled frozen star, revert it completely and rethink

Often use should be Often Used
DO police or swat teams even exist anymore? If not, make it a historical reference